utils/ActiveGames.js

```diff
@@ -0,0 +1,10 @@
+class ActiveGames{
+    add(room_id, room_obj){
```

Comment:
it should initialize an empty dictionary

```
class ActiveGames {
     activeGamesSet = {}


    add ( ) {
    ...
    }
}
```

Reply:
like in here
https://github.com/khalidsharmarke/A-Fake-Artist-Goes-To-New-York/pull/9/files#diff-6ce89e5b021636a347e8d3ed4e43055c9ba2357870bfa913c76c796dd6f68c40R4

Reply:
i disagree. this becomes redundant.
given the scope of the class as an implementation to store rooms in a singular obj
i think it best to avoid having to write ActiveGames.rooms['insert room id here']

Reply:
look up `extends object`